cmake/DockerConfig.cmake

```diff
@@ -65,10 +65,19 @@ add_custom_target(
     COMMAND ${CMAKE_SOURCE_DIR}/docker/DockerBuild.sh
         -u 1000
         -g 1000
+        -t minimal
         -v ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}
-        -i minimal
+        -c ENABLE_PYTHON=OFF
+        -c ENABLE_LIBRDKAFKA=ON
+        -c ENABLE_AWS=ON
+        -c DISABLE_CONTROLLER=ON
+        -c DISABLE_SCRIPTING=ON
+        -c DISABLE_PYTHON_SCRIPTING=ON
+        -c DISABLE_ENCRYPT_CONFIG=ON
+        -c AWS_ENABLE_UNITY_BUILD=OFF
         -c DOCKER_BASE_IMAGE=${DOCKER_BASE_IMAGE}
         -c BUILD_NUMBER=${BUILD_NUMBER}
+        -c CMAKE_BUILD_TYPE=MinSizeRel
```

Comment:
I don't think a minimal image should mean size-optimized. I'd rather have inlining unless it's about as embedded as an arduino.

Reply:
Actually the minimal image was at least party about size optimization. It was meant for use cases in cloud environments where MiNiFi is used as a sidecar for every Kubernetes pod for log collection. In this case we only wanted to have a very small image and only include what is necessary for this use case as sidecards should be minimal in size if they are used in every pod. It's not a huge difference if we use the Release build type instead of MinSizeRel as it's 33MB or 39.2MB image though. What would be the benefit of the Release build in this use case?

Reply:
I agree with the first part. By size-optimized, I meant `-Os`. In some cases it gives up inlining, which would enable many other optimizations, loop unrolling and a bunch of other compiler optimizations that increases the size of the binary. I would only go that route if there is a constraint forcing us to do so.

Reply:
Thanks, I see what you mean. As we don't have specific size requirements and as I think under 39.2MB size is still okay, I changed the minimal image to use the `Release` build for those benefits in 2192050b5ae17cbad8c4c700fb072e53c615c644